dask/array/core.py

```diff
@@ -3757,12 +3757,23 @@ def concatenate3(arrays):
            [1, 2, 1, 2],
            [1, 2, 1, 2]])
     """
+    from .utils import IS_NEP18_ACTIVE
+
     arrays = concrete(arrays)
     if not arrays:
         return np.empty(0)
 
     advanced = max(core.flatten(arrays, container=(list, tuple)),
                    key=lambda x: getattr(x, '__array_priority__', 0))
+
+    if (IS_NEP18_ACTIVE and not all(np.ndarray.__array_function__ is
+                                    getattr(arr, '__array_function__', np.ndarray.__array_function__) for arr in arrays)):
```

Comment:
```suggestion
                                    getattr(arr, '__array_function__', np.ndarray.__array_function__)
                                    for arr in arrays)):
```

Reply:
@hameerabbasi sorry, looks like lint took the week off to pester you. This should fix it. :)

Reply:
Weird... I applied autopep8